tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@salesforce/dev-config/tsconfig",
   "compilerOptions": {
     "outDir": "lib",
+    "allowSyntheticDefaultImports": true,
```

Comment:
wasn't able to build the plugin without this 🤷🏼 
https://github.com/TooTallNate/node-agent-base/issues/56